webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,49 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (userSpecificMetadata) {
+        if (
+            userSpecificMetadata != null &&
+            userSpecificMetadata !== 'undefined'
+        ) {
+            var [groupMetadata, metadataToShow] = [
```

Comment:
what's the benefit of combining it one line using [. ]
this way it seems difficult to understand the code

Reply:
updated